docs/content/docs/dev/datastream/operators/windows.md

```diff
@@ -557,6 +557,31 @@ input
     .aggregate(new AverageAggregate)
 ```
 {{< /tab >}}
+{{< tab "Python" >}}
```

Comment:
should also update the chinese doc: docs.zh/content/docs/dev/datastream/operators/windows.md